msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -199,6 +202,57 @@ public AcquireTokenResult acquireTokenSilent(AcquireTokenSilentOperationParamete
         }
     }
 
+    /**
+     * Get MSAL PublicClientApplication mode from Broker.
+     * */
+    public void getBrokerAccountMode(final Context appContext,
+                                     final PublicClientApplication.BrokerAccountModeCallback callback) {
+
+        final String methodName = ":getBrokerAccountMode";
+        final Handler handler = new Handler(Looper.getMainLooper());
+
+        new Thread(new Runnable() {
```

Comment:
Rather than make a one-off thread, can we add an `Executor` to this class and submit the job?